testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -455,45 +444,35 @@
 
 Then(/^I should see "([^"]*)" systems selected for SSM$/) do |arg|
   within(:xpath, '//span[@id="spacewalk-set-system_list-counter"]') do
-    raise unless has_content?(arg)
+    raise "Systems #{arg} not selected" unless has_content?(arg)
```

Comment:
I wanted to say this should probably be `#{arg} systems not selected`, like in `5 systems not selected`. If this is realy a number, `Systems 5 not selected` is not a proper error message.

Reply:
done